lib/test/nightmare-service.js

```diff
@@ -51,6 +52,20 @@ module.exports = class NightmareService {
 
   prepareTestArgs(options) {
     const serveUrl = `http://${options.host || 'localhost'}:${options.port || '3000'}`;
+    // send report command first to produce coverage
+    this.nycArgs = ['report',
```

Comment:
It may be appropriate to relocate these as well into something like, "prepareCoverageArgs()".  I see "prepareTestArgs" as being concerned with ui-testing's "test-module" only.  This makes testing the function more clear.